### DISCUSSION TITLE
System Capabilities

### DISCUSSION THREAD

**Discussion by Pradyy** [lldpd/lldpd#507] [Q&A]
Is there a way to configure what System Capabilities we advertize? For example can we set Station Only and not advertize any other Capabilities?

The lldp Spec says this in the System Capabilities section:
<<<
The Station Only capability is intended for devices that implement only an end station
capability, and for which none of the other capabilities in the table apply. Bit 8 should
therefore not be set in conjunction with any other bits.  
>>>

But my advertized TLV looks like this in debian:
	System Capabilities TLV (7), length 4
	  System  Capabilities [Bridge, WLAN AP, Router, Station Only] (0x009c)
	  Enabled Capabilities [Station Only] (0x0080)

ANy idea why it set Bit 8 in conjunction with other bits?

**Comment by vincentbernat** [comment]
This is not configurable. This is often asked, but I never got around implementing ability to override what's detected.

As for station only being set in conjunction with other bits, this seems simple enough to fix. Do you care to propose a patch?
